app/javascript/app/components/emission-pathways-table-wrapper/emission-pathways-table-wrapper-component.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import AnchorNav from 'components/anchor-nav';
+import layout from 'styles/layout.scss';
+import Button from 'components/button';
+import cx from 'classnames';
+import anchorNavLightTheme from 'styles/themes/anchor-nav/anchor-nav-light.scss';
+import styles from './emission-pathways-table-wrapper-styles.scss';
+
+class EmissionPathwaysTableWrapper extends PureComponent {
```

Comment:
As you aren't actually placing the graph inside this component can we change the name from 'wrapper' ?. Maybe EmissionPathwaysTableMenu instead?